flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -144,7 +146,7 @@ public void testKeyedValueStateSerializerReconfiguration() throws Exception {
                         new TestType.ReconfigurationRequiringTestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedValueStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```

Comment:
```suggestion
    void testKeyedValueStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```
Could be package private